app.js

```diff
@@ -88,15 +83,45 @@ const runAll = () => {
   }
 };
 
-// dev mode will watch your conf files and reload everything when they are changed:
-if (argv.mode === 'dev' || argv._.dev || argv._.indexOf('dev') > -1) {
+const loadConfig = () => {
+  return require('confi')({
+    path: [
+      defaultConf,
+      argv.config
+    ],
+    context: {
+      CKDIR: __dirname
+    }
+  });
+};
+const runDev = () => {
   const watchedConfigFiles = [
     path.join(defaultConf, '*'),
     path.join(argv.config, '*'),
   ];
   watcher(watchedConfigFiles, [''], () => {
     runAll();
   }, 100);
+};
+
+const evalCss = (cssExpression) => {
+  // todo: probably need to actually eval the css expression
```

Comment:
the @mixin was just an example, I want to eval css.  We probably should read css from stdin.  like `echo some.css | clientkit --css`